### ISSUE TITLE
Outstanding issues for HDX streamline features

### ISSUE THREAD

**Issue by broncio123** [ococrook/hdxstats#38]
Loose ends (bugs and potential enhancements)

- [x] Find out why Manhattan plots fail for Case Study No. 3
https://github.com/ococrook/hdxstats/blob/db12beb29f13262013ec4059c6025f74dfc2cf36/vignettes/tutorial-streamline.Rmd#L369-L372
. Assign: @broncio123 

- [x] Find out if `colData(qDF)` and `rowData(qDF)` can be implemented to store peptide `Start` and `End` residue numbers, instead of `qDF@metadata`. Assing: @broncio123 
- [x] Replace some methods with more robust versions of them, e.g., `seq_len(5)` instead of `1:5`. Assign: @ococrook (@broncio123 can do too, but not very familiar with what methods need replacement)
- [x] #40
- [x] Unit tests. Assign: @ococrook
- [x] Use `rlog` functions in `app_v2` module to handle logs more graciously and direct log history to file. Check [this](https://rlog.sellorm.com/). Assing: @broncio123 
- [ ] #41
- [ ] Add examples of use in docstrings of streamline backend functions. Assign: @broncio123 
- [ ] Find out why colouring is inconsistent between Protection/Deprotection heatmap and PDB viewer. Assign: @broncio123 

- [x] Specify parent module for those methods with identical names in other modules. Assign: @broncio123

**Comment by broncio123** [comment]
@ococrook feel free to edit the list above.
I probably forgot I few more we talked about.

**Comment by ococrook** [comment]
`extract_hdx_data` fails if there spaces in csv titles, because when R loads them it will replaces spaces with a `.`. Do we tell people to format their csvs correctly or  handle this?

**Comment by broncio123** [comment]
> 

Yes, sounds sensible to ask users to at least format their titles with underscores. 
However, I can add a check to spot those cases to raise an error whenever the number of titles and data cols don't match.
